docs/state-committees/run-node/configuration.md

```diff
@@ -25,22 +25,50 @@ The `config.toml` file in the repository contains various fields that should be
 
 - `ConcurrentFetchers`: This parameter can be used to control the number of parallel fetchers that are responsible to fetch a block. Default value is `8`.
 
-- `OperatorKeystorePath` = EigenLayer operator keystore file path
+- `SignerServerURL`: This is the gRPC server URL for signer for secure key management.
 
-- `OperatorKeystorePasswordPath` = EigenLayer operator keystore password file path
+- `OperatorAddress`: EigenLayer operator address
 
-- `SignerECDSAKeystorePath` = ECDSA signer keystore file path
+- `AccountID`: Account ID for setting up signer for your keys. The operator can choose the account id of their choice.
```

Comment:
what is `AccountID`?

Reply:
https://github.com/Lagrange-Labs/lsc-client-cli/blob/develop/config_signer.toml#L11